src/frontend/mosh-server.cc

```diff
@@ -657,6 +664,12 @@ static void serve( int host_fd, Terminal::Complete &terminal, ServerConnection &
   socklen_t saved_addr_len = 0;
   #endif
 
+  #ifdef HAVE_SYSLOG
+  Addr syslog_saved_addr;
+  socklen_t syslog_saved_addr_len = 0;
+  openlog("mosh-server", LOG_PID | LOG_NDELAY, LOG_AUTH);
```

Comment:
`argv[0]`, maybe?

Also, please move the `openlog()` to `run_server()`, before the `pledge()` call.  That's the break between normal operation and sandboxed operation.  (It doesn't actually matter for OpenBSD, which now has a `sendsyslog()` syscall to handle this and other issues, but the principle remains, for future sandboxing efforts.)

Reply:
Ok will fix this.